kvbc/include/reconfiguration_kvbc_handler.hpp

```diff
@@ -209,6 +204,11 @@ class InternalKvReconfigurationHandler : public concord::reconfiguration::IRecon
               uint32_t,
               const std::optional<bftEngine::Timestamp>&,
               concord::messages::ReconfigurationResponse&) override;
+  bool handle(const concord::messages::ReplicaTlsExchangeKey&,
+              uint64_t,
+              uint32_t,
```

Comment:
The whole file is skipping the second and third parameter but the semantics are unclear (at least from just looking at this function signature). Would you mind adding variable names?

Reply:
This should be fixed, but not as part of this PR